pkg/driftctl.go

```diff
@@ -136,6 +142,13 @@ func (d DriftCTL) Run() (*analyser.Analysis, error) {
 	analysis.Duration = time.Since(start)
 	analysis.Date = time.Now()
 
+	d.s.Bucket(memstore.TelemetryBucket).Set("version", version.Current())
```

Comment:
We can set theses values earlier, imagine one day if we want to send telemetry even if a scan fail, we should be able to send all telemetry data not related to scan result in any way. Btw, we do not need to pass the memstore to driftctl IMHO, we can set telemetry keys related to analysis results in the called code (`scan.go`).